### DISCUSSION TITLE
How to display page url with domain name in dashboard

### DISCUSSION THREAD

**Discussion by ghost** [plausible/analytics#1851] [Anything Else]
In the "Page url" area, the url in "/xxx.html" format is the default.
How to display "Page url" containing domain name, for example: "http(s)://xxx.com/xxx.html"

**Comment by metmarkosaric** [comment]
that's not possible at the moment unfortunately. we keep the domain name hidden in order to save space so it's easier see the page URL itself. is there a specific reason why you'd like to see the domain name in the pages list?

**Reply by ghost** [reply]
> that's not possible at the moment unfortunately. we keep the domain name hidden in order to save space so it's easier see the page URL itself. is there a specific reason why you'd like to see the domain name in the pages list?

Yes, I use self hosting.
If I have 100 domains or more and add them to the dashboard, that would be a huge workload and performance overhead.
So, I just create a website stat and name it "overview" and use 'data-domain="overview"' in javascript to send traffic.
I can get stats for all sites. But I can't tell which domain name they are from.
